src/controllers/settings_controller.js

```diff
@@ -21,6 +21,7 @@ const fs = require('fs');
 const TunnelService = require('../ssltunnel');
```

Comment:
These changes seem to be unrelated to the adapter changes. Please move to a separate PR

Reply:
I had planned on it, but this PR disables all plugins by default, so the gateway would be put into a useless state without the enablement code in place.

Reply:
I'd probably do this in 2 stages. Have ZigBee and ZWave still enabled by default - they will fail to load if the dongles are missing anyways - and then later on change zigbee and zwave to be off by default.

I guess I was seeing a bunch of changes related to cerbot and stuff and didn't understand how that fit in with the adapter changes.